owl/EASE-OBJ.owl

```diff
@@ -1599,6 +1631,31 @@
     
 
 
+    <!-- http://www.ease-crc.org/ont/EASE-OBJ.owl#Feature -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/EASE-OBJ.owl#Feature">
+        <rdfs:subClassOf rdf:resource="http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#Object"/>
+        <rdfs:subClassOf>
+            <owl:Restriction>
+                <owl:onProperty rdf:resource="http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#hasPart"/>
+                <owl:allValuesFrom rdf:resource="http://www.ease-crc.org/ont/EASE-OBJ.owl#Feature"/>
+            </owl:Restriction>
+        </rdfs:subClassOf>
+        <rdfs:subClassOf>
+            <owl:Restriction>
+                <owl:onProperty rdf:resource="http://www.ease-crc.org/ont/EASE-OBJ.owl#isFeatureOf"/>
+                <owl:qualifiedCardinality rdf:datatype="http://www.w3.org/2001/XMLSchema#nonNegativeInteger">1</owl:qualifiedCardinality>
+                <owl:onClass rdf:resource="http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#PhysicalObject"/>
+            </owl:Restriction>
+        </rdfs:subClassOf>
+        <owl:disjointWith rdf:resource="http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#Agent"/>
+        <owl:disjointWith rdf:resource="http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#PhysicalObject"/>
```

Comment:
I guess this is a bit questionable. Let's consider the sticker on the footer of our retail shelves. It seems that the sticker is both a physical object that is a component of the shelve, but also a feature that is useful to detect a shelve.

Reply:
We may consider some refactoring at some point. I think your idea to have features as constituents, rather than parts, is because the features do not exist independently of the whole. (A hole in the cheese can't exist without the cheese).

Under this logic, the sticker is a part that can have independent existence.

Perhaps both parts and features can be information bearing, and the borne information aids detection.